rep-2004.rst

```diff
@@ -636,43 +636,43 @@ Quality Declaration Template
 
   Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level N in REP-2004](https://www.ros.org/reps/rep-2004.html).
 
-  ## Version Policy
+  ## [1] Version Policy
 
-  ### Version Scheme
+  ### [1.i] Version Scheme
```

Comment:
I think it could be valid to add a section for `1.ii` (stable version), where packages could explain something like "This package's first stable release was version X.X.X, released on 202X-XX-XX." with a link to the release notes / binaries.